packages/react/src/auto/interfaces/AutoRelationshipInputProps.tsx

```diff
@@ -3,5 +3,10 @@ import { Control } from "react-hook-form";
 export interface AutoRelationshipInputProps {
   field: string;
   control?: Control<any>;
-  optionLabel?: string | ((record: Record<string, any>) => string);
+  optionLabel?: OptionLabel;
 }
+
+/**
+ * Type for the option label when displaying a list of records from a related model
+ */
+export type OptionLabel = string | ((record: Record<string, any>) => string);
```

Comment:
```suggestion
export type OptionLabel = ReactNode | ((record: Record<string, any>) => ReactNode);
```

so folks can pass styled elements from the design system or what have you

Reply:
Updated to be a node. Good call 👍